modular_skyrat/modules/contractor/code/datums/contractor_support.dm

```diff
@@ -0,0 +1,48 @@
+/datum/antagonist/traitor/contractor_support
+	name = "Contractor Support Unit"
+	antag_moodlet = /datum/mood_event/focused
+
+	show_in_roundend = FALSE // We're already adding them in to the contractor's roundend.
+	give_objectives = TRUE // We give them their own custom objective.
+	show_in_antagpanel = FALSE // Not a proper/full antag.
+	give_uplink = FALSE // Don't give them an uplink.
+	/// Team datum that contains the contractor and the support unit
+	var/datum/team/contractor_team/contractor_team
+
+/// Team for storing both the contractor and their support unit - only really for the HUD and admin logging.
+/datum/team/contractor_team
+	show_roundend_report = FALSE
+
+/datum/antagonist/traitor/contractor_support/forge_traitor_objectives()
+	var/datum/objective/generic_objective = new
+
+	generic_objective.name = "Follow the Contractor's Orders"
+	generic_objective.explanation_text = "Follow your orders. Assist agents in this mission area."
+
+	generic_objective.completed = TRUE
+
+	objectives += generic_objective
+
+/datum/outfit/contractor_partner
+	name = "Contractor Support Unit"
+
+	uniform = /obj/item/clothing/under/chameleon
+	suit = /obj/item/clothing/suit/chameleon
+	back = /obj/item/storage/backpack
+	belt = /obj/item/pda/chameleon
+	mask = /obj/item/clothing/mask/cigarette/syndicate
+	shoes = /obj/item/clothing/shoes/chameleon/noslip
+	ears = /obj/item/radio/headset/chameleon
+	id = /obj/item/card/id/advanced/chameleon
+	r_hand = /obj/item/storage/toolbox/syndicate
+	id_trim = /datum/id_trim/chameleon/operative
+
+	backpack_contents = list(/obj/item/storage/box/survival, /obj/item/implanter/uplink, /obj/item/clothing/mask/chameleon,
+							/obj/item/storage/fancy/cigarettes/cigpack_syndicate, /obj/item/lighter)
```

Comment:
please make the items in the list on separate lines each--
item1,
item2,
item3
not item1, item2, item3